lunar-install/sbin/lunar-install

```diff
@@ -1102,14 +1102,17 @@ transfer()
 
     (
       percent_msg "Creating base LSB directories"
-      mkdir -p bin boot dev etc home lib mnt media
-      mkdir -p proc root sbin srv tmp usr var opt
+      mkdir -p boot dev etc home mnt media
+      mkdir -p proc root srv tmp usr var opt
       mkdir -p sys
       if [ `arch` == "x86_64" ]; then
         ln -sf /lib lib64
```

Comment:
Let's point this directly to /usr/lib as well, no point having a symlink pointing to another symlink :D